src/images/components/milestones/shell.svg

```diff
@@ -0,0 +1,134 @@
+    <path d="M422.994 418.395C418.313 417.995 413.861 417.493 409.568 416.922C391.427 414.456 378.423 410.711 378.743 407.629C379.063 404.546 392.443 403.062 410.71 403.793C414.98 403.964 419.501 404.238 424.182 404.649C448.979 406.795 468.81 411.625 468.479 415.415C468.147 419.205 447.791 420.53 422.994 418.395Z" fill="#BAF7DD"/>
+    <path d="M409.568 416.922C391.427 414.456 378.423 410.711 378.743 407.629C379.063 404.546 392.443 403.062 410.71 403.793C428.874 406.247 441.866 410.003 441.604 413.075C441.341 416.146 427.881 417.63 409.568 416.922Z" fill="#1A1542"/>
+    <path d="M382.144 453.091C381.083 448.524 380.169 444.071 379.393 439.87C376.139 421.854 375.683 408.326 378.697 407.629C381.711 406.933 387.248 419.274 392.237 436.89C393.379 441.012 394.52 445.396 395.582 449.974C401.21 474.223 402.74 494.568 399.007 495.424C395.274 496.28 387.727 477.328 382.144 453.091Z" fill="#BAF7DD"/>
+    <path d="M379.393 439.87C376.139 421.854 375.683 408.326 378.697 407.629C381.711 406.933 387.248 419.274 392.237 436.89C395.513 454.929 395.959 468.446 392.956 469.142C389.953 469.839 384.382 457.509 379.393 439.87Z" fill="#1A1542"/>
+    <path d="M336.524 424.96C340.554 422.528 344.516 420.29 348.249 418.258C364.381 409.593 377.099 404.98 378.697 407.641C380.296 410.301 370.26 419.388 355.053 429.561C351.491 431.924 347.678 434.356 343.636 436.776C322.31 449.62 303.438 457.326 301.474 454.107C299.511 450.887 315.209 437.792 336.524 424.96Z" fill="#BAF7DD"/>
+    <path d="M348.249 418.246C364.381 409.581 377.099 404.969 378.697 407.629C380.296 410.289 370.26 419.388 355.053 429.56C338.91 438.249 326.192 442.838 324.594 440.201C322.995 437.564 333.019 428.453 348.249 418.246Z" fill="#1A1542"/>
+    <path d="M349.185 372.888C352.736 375.959 356.035 379.018 359.175 382.021C372.407 394.682 380.718 405.357 378.686 407.697C376.654 410.038 364.986 403.29 350.532 391.976C347.187 389.328 343.682 386.439 340.143 383.368C321.34 366.997 308.142 351.436 310.631 348.604C313.12 345.773 330.393 356.596 349.185 372.888Z" fill="#497B91"/>
+    <path d="M359.186 381.964C372.418 394.625 380.73 405.3 378.698 407.64C376.665 409.981 364.997 403.233 350.544 391.92C337.289 379.258 328.978 368.572 330.998 366.243C333.019 363.914 344.778 370.627 359.186 381.964Z" fill="#1A1542"/>
+    <path d="M402.627 368.835C400.789 373.162 398.916 377.283 397.078 381.142C389.086 397.628 381.54 408.839 378.686 407.629C375.832 406.419 378.594 393.187 384.931 376.005C386.415 371.997 388.082 367.785 389.92 363.446C399.624 340.533 410.333 323.157 413.895 324.629C417.457 326.102 412.331 345.91 402.627 368.835Z" fill="#497B91"/>
+    <path d="M397.078 381.143C389.086 397.628 381.54 408.839 378.686 407.629C375.832 406.419 378.594 393.187 384.931 376.005C392.922 359.496 400.48 348.297 403.312 349.495C406.143 350.694 403.403 363.937 397.078 381.143Z" fill="#1A1542"/>
+    <defs>
+        <linearGradient id="gradient1" x1="135.196" y1="154.484" x2="261.305" y2="303.451" gradientUnits="userSpaceOnUse">
```

Comment:
Just a note, I changed these manually from `paintXXX_linear` to `gradientXXX` so the colors show up correctly in Chrome. Not sure how it works though.

Link to solution: https://stackoverflow.com/a/60249151